scripts/uberenv/spack_configs/toss_3_x86_64_ib/devtools/packages.yaml

```diff
@@ -92,6 +92,15 @@ packages:
    paths:
       tar: /usr
    buildable: False
+  python:
+    paths:
+      python@3.8.2: /usr/tce/packages/python/python-3.8.2
+    buildable: False
+  py-sphinx:
+    version: [2.2.0]
+    paths:
+      py-sphinx@2.2.0: /usr/tce/packages/python/python-3.8.2/
```

Comment:
👍

Reply:
Right, it'll add ~25m to the merge PR time but the increased frequency will hopefully prevent silent regressions.

Reply:
If we're on a debug partition, that may take us past the one hour time limit. Are we running on batch?

Reply:
I believe so, it's a separate allocation for each stage